### DISCUSSION TITLE
Edit/Remove online map sources

### DISCUSSION THREAD

**Discussion by ramiro-alba** [osmandapp/OsmAnd#13464] [Q&A] [answered]
Hi all,

I  have been adding new online map sources and and would like to remove/edit some of them. Is this possible, and how should I proceed.?

Thanks in advance 
Regards

**Comment by tommycrock** [comment, marked as answer]
Hi there. Go to menu, Download maps, then the Local tab. Scroll past Offline maps to Online and cached tile maps. The three dots to the right of the source give you the options to delete (or edit, or clear the cache of tiles). 
As an aside, Download maps isn't very helpfully named for this. Manage maps  or Manage map sources would be better.

**Comment by ramiro-alba** [comment]
Many thanks. I could not see it
Regards

**Reply by Lee-Carre** [reply]
Please mark tommycrock's response as the answer to this question, for the benefit of others.
